source/extensions/bootstrap/internal_listener/client_connection_factory.h

```diff
@@ -15,6 +15,8 @@ namespace Extensions {
 namespace Bootstrap {
 namespace InternalListener {
 
+#define INTERNAL_CONNECTION_DEFAULT_BUFFER_SIZE (1024 * 1024)
```

Comment:
Per style guide, try to avoid defining macros.
Might be replacable with:
```
namespace {
constexpr uint32_t INTERNAL_CONNECTION...
}
```

It would be better to move it into one of the classes if possible.

Reply:
Let me get rid of this constant. Using the default value of buffer_size_ in case it's not configured in proto.